test/tests/functional/pbs_multi_sched.py

```diff
@@ -579,3 +579,93 @@ def test_run_limts_per_scheduler(self):
         self.server.expect(JOB, {'job_state': 'Q'}, id=jid4)
         jc = "Not Running: User has reached server running job limit."
         self.server.expect(JOB, {'comment': jc}, id=jid4)
+
+    def test_multi_fairshare(self):
```

Comment:
Do you want to add another case where we pass an invalid sched name?

Reply:
I'm a little confused on what you are asking me to test.  Are you asking me to try and create a scheduler with an invalid name?  I have no problem writing that test, but I didn't implement that code.  I'd have to familiarize myself with that portion of the code.  I did the multisched review, so it shouldn't be too hard.

Reply:
I was thinking about an illegal argument to the "-I" parameter. For example, if the scheduler "foo" did not exist then you should see the message "Scheduler foo does not exist" if you pass "-I foo". What do you think?

Reply:
I think I'll skip this test.  It would require me to go outside of PTL's fairshare support and run pbsfs directly.  While I can do it, I'd rather not.